### ISSUE TITLE
Incorrect return type for flush() function

### ISSUE THREAD

**Issue by h** [xnimorz/use-debounce#174]
**Describe the bug**
The type declaration of the onDebouncedCallback `flush()` function claims that the return type is `void`. However, that's incorrect: the actual return type is the same as the callback. (Or `undefined` if the callback has never been called yet.) This causes Typescript errors when using `flush()`.

**To Reproduce**
```js
const foo = useDebouncedCallback(() => 'abc', 500);
foo();
const result: string | undefined = foo.flush();
```

results in:

```
foobar.tsx:31:11 - error TS2322: Type 'void' is not assignable to type 'string | undefined'.

31     const result: string | undefined = foo.flush();
```

**Expected behavior**
Typescript type declaration should match the function's actual type.

**use-debounce version:** 10.0.0

**Comment by h** [comment]
Submitted a fix for review in [PR 173](https://github.com/xnimorz/use-debounce/pull/173)

**Comment by xnimorz** [comment]
Thanks @h !

**Comment by xnimorz** [comment]
Merged, I'll cut a new version shortly

**Comment by xnimorz** [comment]
Published use-debounce@10.0.1
